<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveSettings</class>
 <widget class="QDialog" name="CurveSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curve Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/chart_line_edit.png</normaloff>:/icons/icons/chart_line_edit.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General Settings</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="title"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="mode">
          <item>
           <property name="text">
            <string>XY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XY Errors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Statistical Box</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Histogram</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Style</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Point Shape</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="shape">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dot</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cross</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Plus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Circle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Filled Circle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Square</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Diamond</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Eight-Star</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Triangle Up</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Triangle Down</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Square-Cross</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Square-Plus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Circle-Cross</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Circle-Plus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Peace</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Line Dashes</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="dashes">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>50/50</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="color">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Pick Color</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="draw_lines">
          <property name="text">
           <string>Draw Lines</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Histogram</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Num Bins</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="hist_bins">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="hist_low">
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="hist_high">
          <property name="minimum">
           <double>-100000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Low</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>High</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Data Query</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Note: %jobs will be replaced by the Job-Query.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="query"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="test">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Test Result</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="query_results">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>mode</tabstop>
  <tabstop>draw_lines</tabstop>
  <tabstop>shape</tabstop>
  <tabstop>dashes</tabstop>
  <tabstop>color</tabstop>
  <tabstop>hist_bins</tabstop>
  <tabstop>hist_low</tabstop>
  <tabstop>hist_high</tabstop>
  <tabstop>query</tabstop>
  <tabstop>test</tabstop>
  <tabstop>query_results</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CurveSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CurveSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
